README.md

```diff
@@ -19,6 +19,16 @@ Table of Contents
   * [Documentation](#documentation)
   * [Contribute](#contribute)
 
+# Installation
```

Comment:
I think this part is not needed. We should add the git submodule command to the `make deps` that way all submodules are updated when we build deps.

Reply:
Sure, let me fix it 👍